### PR TITLE
Add Boyd County, KY

### PR DESCRIPTION
**Pull request by jtroe**

### CODE REVIEW

**Comment by iandees** [comment]
Where did this shapefile come from? Can you point directly to the source data rather than converting it?

File: sources/us/ky/boyd.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "country": "us",
+        "state": "ky"
+    },
+    "data": "https://rawgit.com/jtroe/b48bf035ce738222d5267ee38e54df8b/raw/094dd554e8550dcfdb6262583d90e4acc6c29c90/ky-boyd.json",
+    "note": "Shapefile converted to GeoJSON via mapshaper.org",
+    "type": "http",
+    "conform": {
+        "type": "geojson",
+        "number": {
+            "field": "NUMBER"
```
Comment (iandees) [inline sources/us/ky/boyd.json]: This should be `"number": "NUMBER",`. No need to specify a `field` in an object.

File: sources/us/ky/boyd.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        },
+        "street": {
+            "field": "STREET"
```
Comment (iandees) [inline sources/us/ky/boyd.json]: Same here.

**Comment by jtroe** [comment]
@iandees I'm doing some consulting work and received this shapefile from my client, who received it from the county.  Unfortunately, I don't believe they have this file hosted.

**Comment by jtroe** [comment]
@iandees, now using source shapefile as requested.

**Comment by iandees** [comment]
Thanks for making these changes, @jtroe!

I uploaded the shapefile zip to our S3 bucket just in case the gist disappears for whatever reason. Can you please replace your data URL with:

https://s3.amazonaws.com/data.openaddresses.io/cache/uploads/iandees/4c9049/MasterAddresses.zip

Also, do you have permission to share this data? Do we have permission to share it with the world?

**Comment by jtroe** [comment]
@iandees, I did receive permission from the Boyd County 911 office to publish these addresses to OpenAddresses.

**Comment by iandees** [comment]
Great, thanks for your hard work!

**Pull request merged**